### PR TITLE
Port CF documentation to Voltrazord

### PR DESCRIPTION
**Pull request by ascott1**
This updates the site to use the latest version of Capital Framework:

- Drop bower
- Use npm to bring in the latest CF components
- Move from Grunt to Gulp
- Mirror the build process used by the CF generator
- Ensure that Jekyll ignores files/directories not needed for the site
- Change Jekyll's highlighter to `rouge` to match the latest GitHub pages

## Preview

<https://ascott1.github.io/capital-framework/>

## Review

Any two of the following: @contolini @Scotchester @KimberlyMunoz @jimmynotjim @cfarm

### CODE REVIEW

File: gulp/tasks/browserSync.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+var gulp = require( 'gulp' );
+var util = require( 'gulp-util' );
+var browserSync = require( 'browser-sync' );
+
+gulp.task( 'browserSync', function() {
+  var port = util.env.port || '4000';
+  browserSync.init( {
+    proxy: 'localhost:' + port
+  } );
+} );
```
Comment (contolini) [inline gulp/tasks/browserSync.js]: Given that this is a jekyll project, are the browser sync and watch tasks needed?

**Comment by jimmynotjim** [comment]
Should the links here be updated? https://ascott1.github.io/capital-framework/components/

**Comment by ascott1** [comment]
@jimmynotjim Those links should be updated to the content generated from usage.md eventually. I think those are the most recent links to use for now, from #266.

File: gulp/tasks/browserSync.js (hunk 1 above)
Comment (ascott1) [inline gulp/tasks/browserSync.js]: I think we make use of some of the watch tasks, I'd be all for taking browsersync out here.

**Comment by jimmynotjim** [comment]
Ah ok, thought that linking to the current, but unfortunately empty, usage docs was included in this PR.

**Comment by jimmynotjim** [comment]
The [Readme](https://github.com/cfpb/capital-framework/blob/16372f658a221110a69d0ba5daf6f55b7cb50857/README.md) needs to be updated for new tooling.

**Comment by jimmynotjim** [comment]
I had trouble getting it going, but adding a Gemfile that matched the Design Manual got it up and running like a champ

**Comment by ascott1** [comment]
I pushed a couple of commits that should address the things above:

- Updates the README
- adds the `github-pages` gem so that locally we have the same dependencies and versions as GitHub
- Simplifies the running of the site. Now just type `npm start` to run the jekyll server and start `gulp watch`
- @contolini I kept browsersync as i think the live reloading when styling things might be nice with the above workflow

@jimmynotjim and @contolini can you review these changes? Thanks!

**Comment by jimmynotjim** [comment]
Works locally! Nice updates to the Readme as well. :+1:

**Comment by contolini** [comment]
:shipit:

**Pull request merged**